### ISSUE TITLE
Unhandled Stream error in pipe

### ISSUE THREAD

**Issue by savager** [leboncoin/morphlingjs#6]
trying to apply a test swagger.json gives me Unhandled stream error in pipe. I tested it as valid json so I'm not sure?

github doesn't support json or i'd upload it. lol

**Comment by vroudge** [comment]
Hey!

Can you mail me that? v.roudge@gmail.com

**Comment by savager** [comment]
Yeah, I imagine it may be a bad file, sorry if it is.. I didn’t make it. So, sorry if it’s our fault. Thanks! Cool project

**Comment by vroudge** [comment]
Don't worry. It's still at an early phase project, so we're looking at swaggers to improve it.
